src/app/feedback/feedback.service.spec.ts

```diff
@@ -0,0 +1,98 @@
+import { ActivatedRoute, Router } from '@angular/router';
+import { AngularFireDatabase } from 'angularfire2/database';
+import { BehaviorSubject } from 'rxjs/BehaviorSubject';
+import { FeedbackService } from './feedback.service';
+import { inject, TestBed } from '@angular/core/testing';
+
+const router = {
+  url: 'abc123'
+};
+
+const activatedRoute = {
+  url: new BehaviorSubject<string>('')
+};
+
+const messages = [
+  {
+    name: 'a',
+    href: 'def456',
+    comment: 'blah',
+    email: 'dan@dan.com'
+  },
+  {
+    name: 'b',
+    href: 'abc123',
+    comment: 'blah',
+    email: 'dan@dan.com'
+  },
+  {
+    name: 'c',
+    href: 'abc123',
+    comment: 'blah',
+    email: 'dan@dan.com'
+  }
+];
+
+let messageStream: any = new BehaviorSubject(messages);
```

Comment:
I had to add the 'push' method which is not in BehaviorSubject (next line adds it)